classes/ObjectModel.php

```diff
@@ -1258,7 +1258,7 @@ public function getWebserviceParameters($ws_params_attribute_name = null)
             }
             if (isset($details['validate'])) {
                 $current_field['validateMethod'] = (
-                    array_key_exists('validateMethod', $resource_parameters['fields'][$field_name]) ?
+                    property_exists($resource_parameters['fields'][$field_name], 'validateMethod') ?
```

Comment:
$resource_parameters['fields'][$field_name] is also an array, cannot use property_exists